src/UI/templates/default/Button/button.less

```diff
@@ -36,6 +36,17 @@ input.btn{
       cursor: default;
     }
   }
+  &.engaged {
+    &,
+    &:hover,
+    &:focus,
+    &:active,
+    &.active {
+      background-color: @il-disabled-btn-bg;
+      border-color: @il-disabled-btn-border;
+      color: @il-disabled-btn-color;
+    }
+  }
 }
```

Comment:
And probably some witdh:100% for btn-graphical.

Reply:
@nhaagen Would you check if that works?

Reply:
@nhaagen Did you try this?